sound/soc/sof/control.c

```diff
@@ -104,19 +104,19 @@ int snd_sof_volume_put(struct snd_kcontrol *kcontrol,
 	}
```

Comment:
The commit message is convoluted: 
The return value is passed from firmware IPC instead of pm function success value.

I think you meant "The existing code returns the status of the pm_runtime function but we should return the status of the IPC"

Reply:
Yep, That's much better. Thanks I'll use that.